### PR TITLE
Add KVM support for Congo and Morocco VRB

### PR DESCRIPTION
**Pull request by rajaganeshr**
Added video, vHUB over PCIe nodes for Congo platform.
Added drivers updates for USB gadget, phy and host.

Tested and verified on Congo VRB

### CODE REVIEW

File: drivers/soc/aspeed/Makefile [hunk 1]
```diff
@@ -10,3 +10,6 @@ obj-$(CONFIG_ASPEED_XDMA)		+= aspeed-xdma.o
 obj-$(CONFIG_ASPEED_UDMA)		+= aspeed-udma.o
 obj-$(CONFIG_AST2600_ESPI)              += ast2600-espi.o
 obj-$(CONFIG_AST2700_ESPI)              += ast2700-espi.o
+obj-$(CONFIG_ARCH_ASPEED)		+= aspeed-usb-phy.o
+obj-$(CONFIG_ARCH_ASPEED)		+= aspeed-usb-phy3.o
+obj-$(CONFIG_ARCH_ASPEED)		+= aspeed-usb-hp.o
```
Comment (supven01) [inline drivers/soc/aspeed/Makefile:15]: What is hp?

**Review by supven01 — APPROVED** [review]
Are there any changes that modifies Aspeed's code? if not, Approved.

**Review by supven01 — APPROVED** [review]

**Review by supven01 — APPROVED** [review]

File: drivers/soc/aspeed/Makefile (hunk 1 above)
Comment (rajaganeshr) [reply drivers/soc/aspeed/Makefile:15]: Host controller over PCIe, adding the nodes "usb3ahp"  enables xhci over pci

**Review by mahkurap — APPROVED** [review]

**Comment by ojayanth** [comment]
Looks like Only added Congo dts changes . Required similiar chnages in other PRB?CRB?

**Review by supven01 — APPROVED** [review]

**Pull request merged**